### ISSUE TITLE
cannot retrieve multiple 'set-cookies'

### ISSUE THREAD

**Issue by rizkyputrapb** [jossef/requests#64]
hey, im having a hard time to retrieve multiple 'set-cookies' from my API. as illustrated below, the Postman is receiving 2 'set-cookies' but the Requests can only retrieve 1. is there any workaround on this? thank you!

    Future<LoginModel> login(String email, String password) async {
      final uri = _baseUrl + "/login";
      Map<String, String?> body = {
        'email': email,
        'password': password,
      };
      final response = await Requests.post(uri, body: body, persistCookies: true);
      if (response.success) {
        print("success");
      
        //check for cookies
        var cookie = await Requests.getStoredCookies(Requests.getHostname(uri));
        print("cookie length: $cookie");
        print("cookie user: ${cookie["user"]}");
        print("cookie token_id: ${cookie["token_id"]}");
      
        return LoginModel.fromJson(json.decode(response.content()));
      } else {
        print(response.statusCode);
        response.throwForStatus();
        throw Exception('Failed to login');
      }
    }

```
Future<UserModel> getUserDetail() async {
    final uri = _baseUrl + "/user";
    final response = await Requests.get(uri);
    print(response);
    if (response.success) {
      print("success");
      return UserModel.fromJson(json.decode(response.content()));
    } else {
      print(response.statusCode);
      response.throwForStatus();
      throw Exception('Failed to retrieve user info');
    }
  }
```

![image](https://user-images.githubusercontent.com/43906521/165223724-cea85c70-d074-4fcc-92c7-ec7b662ab6af.png)
![image](https://user-images.githubusercontent.com/43906521/165223764-5d0dece5-2e37-4a18-b6ec-442d6317323d.png)

**Comment by sehnryr** [comment]
I think I know why. The current parsing algorithm of the releases is broken as it does not fully extract the cookies. The issue was that `http` would concatenate all the `SET-COOKIES` headers with `,`. #47 fixed that but I want to wait for #63 to create a new release.

Alternatively you can install the package via git like so:
```yaml
dependencies:
  requests:
    git:
      url: https://github.com/jossef/requests
      ref: master
```

**Comment by sehnryr** [comment]
Normally this issue has be fixed with the new `4.2.0` update.
